app/(golems)/golems/skills/[name]/EvalDashboard.tsx

```
@@ -0,0 +1,655 @@
+}
+
+function PassFailCell({ passed }: { passed: boolean }) {
+  return (
+    <td className="px-3 py-2.5 text-center">
+      {passed ? (
+        <span
+          className="inline-flex h-6 w-6 items-center justify-center rounded-full bg-[#28c84015] text-[#28c840]"
+          aria-label="Passed"
+          title="Passed"
+        >
+          <svg className="h-3.5 w-3.5" viewBox="0 0 16 16" fill="none">
```

Biome (2.4.6)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)